recipes/napari/meta.yaml

```diff
@@ -13,7 +13,6 @@ source:
   sha256: 0e4a20f20be5db9224383e090d64718fb42c5359cdd8ee8be5c01076b7df230d
 
 build:
-  noarch: python
```

Comment:
You can keep this and have a constraint on python version instead of doing skipping.

Reply:
@isuruf great! It felt wrong to remove it. But could you point me to how to do it and where it is documented? The docs I saw all said that if it didn’t run on all the currently supported conda-forge Python versions, I should remove noarch. Happy to contribute an update to the docs if you show me how to update this correctly! Thanks!

Reply:
Docs at https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python is confusing. See https://github.com/conda-forge/awswrangler-feedstock/blob/440b2218dea38355a3314ff12c63c1daedd45e66/recipe/meta.yaml for an example.